src/main/log-handler.ts

```diff
@@ -40,10 +40,10 @@ export function createLogHandler() {
       ...args: any[]
     ) => {
       const timestamp = new Date().toISOString()
-      let line = [timestamp, channel, level]
-      line = line
-        .concat([stacktrace, ...args])
-        .map(value => JSON.stringify(value))
+      let line = [timestamp, fillString(channel, 22), level]
```

Comment:
Why do we need to fill this? Doesn't it look weird?

Reply:
because it makes it more readable in an text editor.